juniper_official/Solutions/EX-Platform/PoE-KPI-Playbook.playbook

```diff
@@ -0,0 +1,19 @@
+/*
+ * Playbook contains 3 rules which monitors POE status , interface and controller 
+ * power consumption and notifies the status.
+ * 
+ * 1) Rule check-poe-total-power, checks POE Controller power consumption and raises 
+ *    alarm in case of anomalies.
+ * 2) Rule check-poe-operational-status, POE Interface Operational state and  
+ *    notifies when anomalies are found.
+ * 3) Rule check-power-consumption, POE Interface Power consumption and notifies 
+ *    when anomalies are found.
```

Comment:
s/POE Interface Power consumption and notifies 
 *    when anomalies are found./ POE Interface Power consumption and notifies 
 *    when  power consumption is above threshold/g

Reply:
updated